src/components/FaqSection.tsx

```diff
@@ -0,0 +1,280 @@
+"use client";
+import { Box, Button, Typography, colors, styled } from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import Link from "next/link";
+
+import MuiAccordion, { AccordionProps } from "@mui/material/Accordion";
+import MuiAccordionSummary, {
+  AccordionSummaryProps,
+} from "@mui/material/AccordionSummary";
+import MuiAccordionDetails from "@mui/material/AccordionDetails";
+
+const Accordion = styled((props: AccordionProps) => (
+  <MuiAccordion disableGutters elevation={0} square {...props} />
+))(() => ({
+  borderTop: 0,
+  borderBottom: "1px solid rgba(0, 0, 0, .125)",
+  width: "100%",
+}));
+
+const AccordionSummary = styled((props: AccordionSummaryProps) => (
+  <MuiAccordionSummary
+    expandIcon={<ExpandMoreIcon sx={{ fontSize: "2rem" }} />}
+    {...props}
+  />
+))(() => ({
+  backgroundColor: "#fcf9f2",
```

Comment:
for background color and color page set on the theme.tsx and use the value from theme property krub
eg. `theme.background.customBackground`